sass/_header.scss

```diff
@@ -3,125 +3,114 @@ Header
 --------------------------------------------------------------*/
 
 .header {
-    @include rem(font-size, 24px);
-    color: #fff;
-	padding: 0 20px;
+  @include rem(font-size, 24px);
+  color: #fff;
+  padding: 0 20px;
+  position: absolute;
+  width: 100%;
+  z-index: $z-index-header;
+
+  .backdrop {
+    background-color: $header-color;
+    bottom: 0;
+    left: 0;
+    opacity: 0;
     position: absolute;
-    width: 100%;
-    z-index: $z-index-header;
-	
-    .backdrop {
-		background-color: $header-color;
-		bottom: 0;
-		left: 0;
-		opacity: 0;
-		position: absolute;
-		right: 0;
-		top: 0;
+    right: 0;
+    top: 0;
+    transition: opacity 200ms;
+    z-index: $z-index-header-backdrop;
+
+    .noscroll & {
+      background-color: transparent !important;
+    }
+  }
+
+  &.bg .backdrop {
+    opacity: 1;
+  }
+
+  a {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.